test/UriTest.php

```diff
@@ -609,6 +609,12 @@ public function utf8QueryStringsDataProvider()
     public function testUriDoesNotAppendColonToHostIfPortIsEmpty()
     {
         $uri = (new Uri())->withHost('google.com');
-        $this->assertEquals('google.com', (string) $uri);
+        $this->assertEquals('//google.com', (string) $uri);
+    }
+
+    public function testAuthorityIsPrefixedByDoubleSlashIfPresent()
+    {
+        $uri = (new Uri())->withHost('example.com');
+        $this->assertEquals('//example.com', (string) $uri);
```

Comment:
This test is exactly the same as `testUriDoesNotAppendColonToHostIfPortIsEmpty` except the URL used... not sure we need both except for being explicit.